code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -287,18 +287,25 @@ \section{Specific System Description}
 \subsubsection{Terminology and Definitions}
 \label{Sec:TermDefs}
 This subsection provides a list of terms that are used in the subsequent sections and their meaning, with the purpose of reducing ambiguity and making it easier to correctly understand the requirements.
 \begin{itemize}
-\item[Factor of Safety:]The global stability of a surface in a slope
-\item[Critical Slip Surface:]Slip surface of the slope that has the lowest global factor of safety, and therefore most likely to experience failure.
+\item[Factor of Safety:]The global stability metric of a slip surface of a slope, defined as the ratio of resistive shear force to mobilized shear force
+\item[Slip Surface:]A surface within a slope that has the potential to fail or displace due to load or other forces.
+\item[Critical Slip Surface:]Slip surface of the slope that has the lowest factor of safety, and is therefore most likely to experience failure.
+\item[Water Table:]The upper boundary of a saturated zone in the ground.
 \item[Stress:]Forces that are exerted between planes internal to a larger body subject to external loading.
-\item[Strain:]Stress forces that result in deformation of the body/plane.
+\item[Strain:]A measure of deformation of a body or plane under stress.
 \item[Normal Force:]A force applied perpendicular to the plane of the material.
 \item[Shear Force:]A force applied parallel to the plane of the material.
-\item[Plane Strain:]The resultant stresses in one of the directions of a 3 dimensional material can be approximated as 0. Results when the length of one dimension of the body dominates the others. Stresses in the dominant dimensions direction are the ones that can be approximated as 0.
+\item[Mobilized Shear Force:]Shear force in the direction of potential motion, thus encouraging motion along the plane.
+\item[Resistive Shear Force:]Shear force in the direction opposite to the direction of potential motion, thus hindering motion along the plane.
+\item[Effective Forces and Stresses:]The normal force or normal stress carried by the soil skeleton. The total normal force or normal stress is composed of the effective force or stress and the force or stress exerted by water.
+\item[Cohesion:]An attractive force between adjacent particles that holds the matter together.
+\item[Isotropy:]A condition where the value of a property is independent of the direction in which it is measured.
+\item[Plane Strain:]A condition where the resultant stresses in one of the directions of a  three-dimensional material can be approximated as zero. Results when the length of one dimension of the body dominates the others, to the point where it can be assumed as infinite. Stresses in the direction of the dominant dimension can be approximated as zero.
```

Comment:
I suggest changing the second sentence to `This condition results when a body is constrained to not deform in one direction, or when the length of one dimension of the body dominates the others, to the point where it can be assumed as infinite.`

Reply:
The requested change has been made in 7e4d9e2.